src/Storages/StorageMergeTree.cpp

```diff
@@ -1240,56 +1240,63 @@ MergeTreeDataPartPtr StorageMergeTree::outdatePart(const String & part_name, boo
     }
 }
 
-void StorageMergeTree::dropPartition(const ASTPtr & partition, bool detach, bool drop_part, ContextPtr local_context, bool throw_if_noop)
+void StorageMergeTree::dropPartNoWaitNoThrow(const String & part_name)
 {
-    {
-        MergeTreeData::DataPartsVector parts_to_remove;
-        auto metadata_snapshot = getInMemoryMetadataPtr();
+    if (auto part = outdatePart(part_name, false))
```

Comment:
```suggestion
    if (auto part = outdatePart(part_name, /* force*/ false))
```
btw that 'force' is a bit misleading...

Reply:
To make [clang-tidy](https://clang.llvm.org/extra/clang-tidy/checks/bugprone-argument-comment.html) complains, it should be

```suggestion
    if (auto part = outdatePart(part_name, /* force= */ false))
```

(Here and below)